_posts/2018-04-11-MealsAndYou.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Meals and You
+date: 2018-4-11
+categories:
+-APIs
+-python Flask
+-javascript
+-bootstrap
+-html
+description: Takes in an ingredient on what the user wishes to cook with and returns a list of meals
+they can make and the ingredients that they must then buy in order to make them, along with the recipe themselves.
+image: n/a
+image-sm: n/a
+author: Nina Wiggins
+author-image: n/a 
+author-bio: First Year Computer Science Major at Rochester Institute of Technology
+author-email: ninawiggins11@gmail.com
+author-social:
+  github: https://github.com/nxw9753 
+---
```

Comment:
Add the blog post content under this `---`, preferably with headers (syntax is `# Header text here`) marking sections.